blazor/diagram/tool-tip.md

```diff
@@ -19,4 +19,656 @@ By default, the diagram displays a tooltip that provides information about the s
+@code
+{
+    DiagramObjectCollection<Node> nodes;
+
+    protected override void OnInitialized()
+    {
+        nodes = new DiagramObjectCollection<Node>();
+        Node node = new Node()
+        {
+            ID = "node1",
+            OffsetX = 250,
+            OffsetY = 250,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip(){Content="NodeTooltip",Template=getContent()},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        nodes.Add(node);
+    }
+    private string getContent()
+    {
+        string content = "<div><p>Product Name : Diagram</p><p>Element: Node</p><p>Content: Node Tooltip <p></p></div>";
+        return content;
+    }
+    //Change Template at run time
+    private void TemplateChange()
+    {
+        nodes[0].Tooltip.Template = "<p>TemplateUpdate</p>";
+    }
+}
+```
+
+The following code example illustrates how to add formatted template content to the tooltip for connectors.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Connector Template" @onclick="TemplateChange" />
+<SfDiagramComponent Width="1000px" Height="500px" Connectors="connector" />
+@code
+{
+    //Defines diagram's connector collection.
+    DiagramObjectCollection<Connector> connector;
+
+    protected override void OnInitialized()
+    {
+        connector = new DiagramObjectCollection<Connector>();
+        Connector connectors = new Connector()
+        {
+            ID = "Connector1",
+            SourcePoint = new DiagramPoint() { X = 500, Y = 500 },
+            TargetPoint = new DiagramPoint() { X = 600, Y = 400 },
+            Tooltip = new DiagramTooltip(){Content="ConnectorTooltip",Template=getContent()},
+            Constraints =  ConnectorConstraints.Default|ConnectorConstraints.Tooltip ,
+        };
+        connector.Add(connectors);
+    }
+   private string getContent()
+    {
+        string content = "<div><p>Product Name : Diagram</p><p>Element: Connector</p><p>Content: Connector Tooltip <p></p></div>";
+        return content;
+    }
+    //Change Template at run time
+    private void TemplateChange()
+    {
+        connector[0].Tooltip.Template = "<p>TemplateUpdate</p>";
+    }
+}
+```
+
+## Tooltip animation
+
+To animate the tooltip, a set of specific animation effects are available, and it can be controlled by using the animation property. The [Animation](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_AnimationSettings) property also allows you to set delay, duration, and various other effects of your choice.
+
+The following code example illustrates how to set animation to the tooltip for nodes.
```

Comment:
The following code example illustrates how to set the animation to the tooltip for the nodes.